<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LaunchDialog</class>
 <widget class="QDialog" name="LaunchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>166</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VR Renderer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/Icons/Data/Icon.ico</normaloff>:/Icons/Data/Icon.ico</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>130</y>
     <width>75</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Data/OK.png</normaloff>:/Icons/Data/OK.png</iconset>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>75</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Data/Cancel.png</normaloff>:/Icons/Data/Cancel.png</iconset>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>8</y>
     <width>250</width>
     <height>114</height>
    </rect>
   </property>
   <property name="title">
    <string>Source configuration</string>
   </property>
   <widget class="QComboBox" name="comboBoxVideoSource">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>24</y>
      <width>142</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string>Cameras</string>
    </property>
    <item>
     <property name="text">
      <string>Cameras</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Test MP4 Video</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelSource">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>24</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Video source:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSize">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>54</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Video size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelX">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>54</y>
      <width>16</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditVideoWidth">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>54</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>0000</string>
    </property>
    <property name="text">
     <string>352</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditVideoHeight">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>54</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>0000</string>
    </property>
    <property name="text">
     <string>288</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelSize_2">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>84</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>UDP robot status port:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxUdpPort">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>84</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;On this UDP port &lt;span style=&quot; font-weight:600;&quot;&gt;robot's status information &lt;/span&gt;(e.g. turtle mode status) will be received.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="value">
     <number>8550</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
